hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerRatisRequest.java

```diff
@@ -87,4 +89,24 @@ public void testRequestWithNonExistentBucket()
           oe.getResult());
     }
   }
+
+  @Test
+  public void testUnknownRequestHandling() throws IOException {
```

Comment:
It is better to test from org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB#processRequest. Need to make sure the case stands for both conditions (non-ratis and ratis).

Reply:
I think test can not be constructed with unknown type, as java will not allow any other command type if not defined in proto. can try setting null command type if work.

Reply:
Hi @sumitagrawl, just to clarify, I think ```OzoneManagerProtocolProtos.Type.TestUnknownCommand``` could be reserved for test purposes,  but ```UNKNOWN_REQUEST``` could be removed as currently there is an existing type ```INVALID_REQUEST ```  with the similar meaning.

Reply:
Sure, will be adding a test utilizing `OzoneManagerProtocolServerSideTranslatorPB#processRequest`, thanks.